.gitignore

```diff
@@ -13,3 +13,4 @@ deployment/local-dev/collector-config.yaml
 local.env
 dump.sql
 test-results/
+frontend/**/*-storageState.json
```

Comment:
I think this is out of date?
```suggestion
```

Reply:
It might be, but I still need to verify that the storageState.json files aren't showing up any more.

Reply:
Removed in commit c0a4b540f0a0197d490857dd45159f67986f8888.